### DISCUSSION TITLE
SNIPE API: Query multiple values for the same parameter

### DISCUSSION THREAD

**Discussion by felixfenu** [snipe/snipe-it#11020] [Q&A] [answered]
Is posibble to ask for two Values for the same argument on SNIPE API?
EG:
https://tenant.snipe-it.io/api/v1/hardware?location_id=11,10
or
https://tenant.snipe-it.io/api/v1/hardware?location_id=11&location_id=10

Thank you very much

**Comment by felixfenu** [comment]
+1

**Comment by snipe** [comment, marked as answer]
Not currently, but we *are* thinking through the best way to handle this moving forward. API design can affect users in the long term, so we have to make sure whatever way we do it, we hopefully don’t have to change it later and mess up people’s integrations.

**Reply by felixfenu** [reply]
Thank you for fast answer!
